packages/augur-ui/src/modules/auth/components/common/hardware-wallet.jsx

```diff
@@ -203,10 +203,7 @@ export default class HardwareWallet extends Component {
           const promises = [];
           walletAddresses.forEach(addr => {
             const getPromise = new Promise((resolve, reject) => {
-              getEtherBalance(addr.address, (err, balance, address) => {
-                if (err) return reject(new Error(err));
-                resolve({ balance, address });
-              });
+              getEthBalance(addr.address).then(balance => resolve({ balance, address })).catch(err => reject(err));
```

Comment:
```getEthBalance``` used to return balance and address now it just returns balance, is that on purpose?

If so, the resolve on line 206 needs to remove the address reference.

Reply:
updated PR, good catch.